Mar12358/JavaScriptCapstone
# Home page and popUp comments window for Meals using APIs

### Added features:
- Added Header
- Added Footer
- Home Page
  - GET elements from Meals API and list them. 
  - POST/GET likes into/from Involvement API
  - Implement the `addLike `button.
  - Count items
  - Test items counter
- Popup comments window
  - Add more details from Meals API
  - POST/GET comments into/from Involvement API
  - Add Comment
  - Count Comments
  - Test comments counter
- Documentation
  - Added README file
  - See the [Presentation Video](https://drive.google.com/file/d/18JcKl66gjpe1g31vRuzaXgKsTXKSL2J0/view?usp=sharing)
- Follow the given design
**Try the [Live Demo](https://mar12358.github.io/JavaScriptCapstone/)**
